invite/forms.py

```diff
@@ -10,20 +10,21 @@ def validate_username(value):
     if value in User.objects.all().values_list('username', flat=True):
         raise ValidationError('Username taken, choose another')
 
-
 def validate_user_email(value):
     insensitive_emails = (
         [e.lower() for e in User.objects.all().values_list('email', flat=True)]
     )
-    assert False, insensitive_emails
     if value.lower() not in insensitive_emails:
         raise ValidationError('The email provided doesn\'t belong to any user')
 
-
 def validate_user_email_exists(value):
-    if value in User.objects.all().values_list('email', flat=True):
+    if value.lower() in User.objects.all().values_list('email', flat=True):
```

Comment:
I wonder if we need to do something like in the above `validate_user_email` function with lowercasing emails that we pull from the User table because those can be created outside of django-invite and may not be guaranteed to be lowercase. The part like:
```
    insensitive_emails = (
        [e.lower() for e in User.objects.all().values_list('email', flat=True)]
    )
```